Exercise 19-JS/index.js

```diff
@@ -0,0 +1,25 @@
+const monthOfBirth = 'June'
+const firstName ='Carlos'
+const lastName ='Horita'
+let city;
+city ='Winnipeg';
+let age;
+age ='35'
+let nextCity;
+nextCity ='Winnipeg'
+
+console.log('monthOfBirth:June, monthOfBirth');
+console.log('firstName:Carlos, firstName');
+console.log('lastName:Horita, lastName');
+console.log('city:Winnipeg, city');
+console.log('age:35, age');
+console.log('nextCity:city, nextCity');
```

Comment:
Everything that goes inside quotes '', will be considered as string. So if you place variable inside too, it will not be considered as variable but as simple text.
should be like this:
console.log('Month of birth:', monthOfBirth);
As you see, only Month of birth text is inside quotes.